gwtbootstrap3/src/main/java/org/gwtbootstrap3/client/ui/base/form/AbstractForm.java

```diff
@@ -455,12 +455,12 @@ private void createFrame() {
     }
 
     /**
-     * Fire a {@link FormPanel.SubmitEvent}.
+     * Fire a {@link AbstractForm.SubmitEvent}.
      *
      * @return true to continue, false if canceled
      */
     private boolean fireSubmitEvent() {
-        FormPanel.SubmitEvent event = new FormPanel.SubmitEvent();
+        SubmitEvent event = new SubmitEvent();
```

Comment:
Do we still need the FormPanel.Submit event then? I will wait for @ArloL to look at this since he overhauled this class recently.